dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+# This script is expected to be run with a PWD of the root of the puppet-server
+# project.
+
+
+# Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+# Legacy means pre-aio (4.0) in this case
+export PUPPET_LEGACY_VERSION='3.7.5'
+
+  # --post-suite acceptance/suites/post_suite_backwards_compatibility \
+bundle exec beaker \
+  --debug \
+  --root-keys \
+  --no-color \
+  --repo-proxy \
+  --preserve-hosts never \
+  --type aio \
+  --config acceptance/config/beaker/jenkins/redhat7-64m-64a.yaml \
+  --pre-suite acceptance/suites/pre_suite/puppet3_compat \
+  --tests acceptance/suites/puppet3_tests \
+  --keyfile ~/.ssh/id_rsa-acceptance \
```

Comment:
You have it correct Nate, the coordinator nodes that Jenkins uses to execute beaker processes already have the identity file on them.  Devs running locally need to get the private key "somehow" which has most commonly taken the form of asking someone who already copied it.